src/inputs/Table/Table.js

```diff
@@ -168,21 +173,46 @@ export const Table = (props) => {
           buildErrorsCountLabel={buildErrorsPanelTitle}
         />
       )}
-      <div data-cy="grid" id="grid-container" style={dimensions} className={agTheme}>
+      <div data-cy="grid" className={agTheme}>
         {isReady && (
-          <AgGridReact
-            ref={gridRef}
-            undoRedoCellEditing={true}
-            rowDragManaged={true}
-            suppressDragLeaveHidesColumns={true}
-            allowDragFromColumnsToolPanel={true}
-            columnDefs={formattedColumns}
-            defaultColDef={defaultColDef}
-            columnTypes={columnTypes}
-            rowData={rows}
-            context={context}
-            stopEditingWhenCellsLoseFocus={true}
-          />
+          <>
+            <TableFilterRow isFullscreen={isFullscreen} setFullscreen={onClickFullscreenButton} />
+            <Box sx={dimensions}>
+              <AgGridReact
+                ref={gridRef}
+                undoRedoCellEditing={true}
+                rowDragManaged={true}
+                suppressDragLeaveHidesColumns={true}
+                allowDragFromColumnsToolPanel={true}
+                columnDefs={formattedColumns}
+                defaultColDef={defaultColDef}
+                columnTypes={columnTypes}
+                rowData={rows}
+                context={context}
+                stopEditingWhenCellsLoseFocus={true}
+              />
+            </Box>
+            <Dialog fullScreen open={isFullscreen} onClose={onClickFullscreenButton} className={agTheme}>
+              <Box p={4} sx={{ height: '100%' }}>
+                <TableFilterRow isFullscreen={isFullscreen} setFullscreen={onClickFullscreenButton} />
+                <Box sx={{ height: 'calc(100% - 32px)' }}>
```

Comment:
Where do the `32px` come from? If it comes from a style defined by us somewhere, we should replace it by a const to make sure both parts of code are always up-to-date :wink:
If it doesn't, could you add a comment to explain where this value comes from please?

Reply:
@csm-thu : those 32px come from the `<TableFilterRow>` 's height: AgGrid starts with a `<div style="height=100%">`, but everything gets pushed down because of this `<TableFilterRow>` and a scrollbar appears, ruining the full-screen.